lessons/pydata/api/index.ipynb

```diff
@@ -0,0 +1,1713 @@
+{
```

Comment:
<p>"libovolná datová struktura" mi úplně nezní, resp. nevím jak je to přesně myšleno. Má se tady říkat, že JSON podporuje tyto datové typy?</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PyDataCZ/naucse.python.cz/pull/32/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
Ten obrat je odsud https://books.google.cz/books?id=3bdgAgAAQBAJ&pg=PA176&lpg=PA176&dq=vstupem+je+datov%C3%A1+struktura+api&source=bl&ots=RE_mlYbV2A&sig=ACfU3U3EOEX0VXM5Pjy5mECmyxRgaYJ13w&hl=cs&sa=X&ved=2ahUKEwi3xrXbqYXnAhVQKFAKHTFODvMQ6AEwAXoECAoQAQ#v=onepage&q=vstupem%20je%20datov%C3%A1%20struktura%20api&f=false co je JSON.

Doplnil jsem, co je výstupem. Může být?

Reply:
<p>ok, je to spíš detail :)</p>